testing/test_pdb.py

```diff
@@ -688,6 +690,23 @@ def test_pdb_custom_cls(self, testdir, custom_pdb_calls):
         result.stdout.fnmatch_lines(["*NameError*xxx*", "*1 error*"])
         assert custom_pdb_calls == ["init", "reset", "interaction"]
 
+    def test_pdb_custom_cls_invalid(self, testdir):
+        result = testdir.runpytest_inprocess("--pdbcls=invalid")
+        result.stderr.fnmatch_lines(
+            [
+                "*: error: argument --pdbcls: 'invalid' is not in the format 'modname:classname'"
+            ]
+        )
+
+    def test_pdb_validate_usepdb_cls(self, testdir):
+        assert _validate_usepdb_cls("os.path:dirname.__name__") == "dirname"
+
+        with pytest.raises(argparse.ArgumentTypeError) as exc:
```

Comment:
Just a reminder that you can use `match=` here 😉

`match` will accept a regex, which you can use to make the test to pass on py27